### PR TITLE
UI for Application Level Mutual Tls Support Feature

### PR DESCRIPTION
**Pull request by Abirame-ship-it**

### CODE REVIEW

**Comment by sajinieKavindya** [comment]
Shall we add a detailed description to this PR and tag the issues (if any) that this PR fixes?

File: portals/devportal/source/src/app/components/Applications/Details/Certificates.jsx [hunk 1]
```diff
@@ -0,0 +1,373 @@
+import React, { useEffect, useState } from 'react';
```
Comment (madushadhanushka) [inline portals/devportal/source/src/app/components/Applications/Details/Certificates.jsx:1]: Lets add licence header

File: portals/devportal/source/src/app/components/Applications/Details/MutualTLS.jsx [hunk 2]
```diff
@@ -0,0 +1,234 @@
+/*
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (madushadhanushka) [inline portals/devportal/source/src/app/components/Applications/Details/MutualTLS.jsx:2]: Change the licence header year to 2022

File: portals/devportal/source/src/app/components/Applications/Details/UploadCertificate.jsx [hunk 3]
```diff
@@ -0,0 +1,296 @@
+import React, { useState } from 'react';
```
Comment (madushadhanushka) [inline portals/devportal/source/src/app/components/Applications/Details/UploadCertificate.jsx:1]: Add licence headers here as well

File: portals/devportal/source/src/app/components/Applications/Details/Certificates.jsx [hunk 4]
```diff
@@ -0,0 +1,373 @@
...
+    const classes = useStyles();
+
+    /**
+     * Show the selected certificate details in a popover.
+     *
+     * @param {any} event The button click event.
+     * @param {string} UUID  The UUID of the certificate which information is required.
+     * */
+    const showCertificateDetails = (event, name, UUID) => {
+        console.log(UUID);
```
Comment (madushadhanushka) [inline portals/devportal/source/src/app/components/Applications/Details/Certificates.jsx:108]: Do we need this logs in production?

**Comment by madushadhanushka** [comment]
Lets trigger another Github actions build and make sure it getting passed

**Pull request closed without merging**